tests/e2e/machine_pool_test.go

```diff
@@ -107,5 +107,115 @@ var _ = Describe("TF Test", func() {
 
 			})
 		})
+		Context("Author:amalykhi-Critical-OCP-68296 @OCP-68296 @amalykhi", func() {
+			It("Author:amalykhi-Critical-OCP-68296 Enable/disable/update autoscaling for additional machinepool", ci.Day2, ci.Critical, ci.FeatureMachinepool, func() {
+				By("Create additional machinepool with autoscaling")
+				replicas := 9
+				minReplicas := 3
+				maxReplicas := 6
+				machineType := "r5.xlarge"
+				name := "ocp-68296"
+				MachinePoolArgs := &exe.MachinePoolArgs{
+					Token:              token,
+					Cluster:            clusterID,
+					MinReplicas:        minReplicas,
+					MaxReplicas:        maxReplicas,
+					MachineType:        machineType,
+					Name:               name,
+					AutoscalingEnabled: true,
+				}
+
+				err := mpService.Create(MachinePoolArgs)
+				Expect(err).ToNot(HaveOccurred())
+				By("Verify the parameters of the created machinepool")
+				mpResponseBody, err := cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, name)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(mpResponseBody.Autoscaling().MinReplicas()).To(Equal(minReplicas))
+				Expect(mpResponseBody.Autoscaling().MaxReplicas()).To(Equal(maxReplicas))
+				By("Change the number of replicas of the machinepool")
```

Comment:
can we do spaces between `By()` to `By()`? just for visibility of the steps taken